app/models/wizards/memberships/terminate_sac_membership_wizard.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module Wizards::Memberships
+  class TerminateSacMembershipWizard < Wizards::Base
+    self.steps = [
+      Wizards::Steps::MembershipTerminatedInfo,
+      Wizards::Steps::AskFamilyMainPerson,
+      Wizards::Steps::TerminationNoSelfService,
+      Wizards::Steps::TerminationChooseDate,
+      Wizards::Steps::Termination::Summary
+    ]
+
+    attr_reader :person
+
+    def initialize(current_step: 0, person: nil, backoffice: nil, **params)
```

Comment:
Maybe we should remove the default values for these parameters. I don't think they make much sense (Probably better in a follow up PR for all wizards.)

Reply:
good idea